site/ko/blog/new-in-devtools-99/index.md

```diff
@@ -6,188 +6,180 @@ authors:
 date: 2022-02-21
 updated: 2022-02-21
 description:
-  "Throttling WebSocket requests, new Reporting API pane, console styling and more."
+  "웹 소켓 요청 쓰로틀링, 새로운 Reporting API 창, console 스타일링 외"
 hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/iStno9LHU3f6DJe6hOIb.jpg'
 alt: ''
 tags:
   - new-in-devtools
   - devtools
   - chrome-99
-draft: true
 ---
 
-*이 게시글의 번역에는 [최원영](https://www.linkedin.com/in/toruchoi)님이 참여하였으며, [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님이 리뷰를 맡아 주셨습니다.*
+*이 게시글의 번역에는 [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님이 참여하였으며, [최원영](https://www.linkedin.com/in/toruchoi)님과 [도창욱](https://developers.google.com/community/experts/directory/profile/profile-changwook-doh)님이 리뷰를 맡아 주셨습니다.*
 
 {% include 'partials/devtools/ko/banner.md' %}
 
-<!-- start: translation instructions -->
-<!-- 1. Remove the "draft: true" tag above when submitting PR -->
-<!-- 2. Provide translations under each of the English commented original content -->
-<!-- 3. Translate the "description" tag above -->
-<!-- 4. Translate all the <img> alt text -->
-<!-- 5. Update the whats-new.md file -->
```

Comment:
Please update the `/ko/whats-new.mdwhats-new.md` file as well.

Reply:
Ahh- I missed it. I'll add it tomorrow